docs/tutorial.rst

```diff
@@ -98,6 +98,74 @@ before copying the data over:
 Note that to ensure consistency of operations, the ``pgcopydb snapshot``
 command has been used. See :ref:`resuming_operations` for details.
 
+Use pgcopydb to migrate from a Citus database to a Citus database
+-----------------------------------------------------------------
+
+It is possible to use the previous tutorial to implement support for
+migrating a Citus database to its new hosting environment. For that, the
+``schema-changes.sql`` script from the previous section needs to be the
+Citus script that calls into the `Citus DDL`__ functions.
+
+__ https://docs.citusdata.com/en/v6.0/dist_tables/ddl.html
+
+Here is an SQL query that exports the commands to use to reproduce a Citus
+distribution scheme:
+
+.. code-block:: sql
+  :linenos:
+   
+   with citus_tables AS
+   (
+     SELECT logicalrelid AS table_name,
+            CASE WHEN colocationid IN (SELECT colocationid FROM pg_dist_schema)
+                 THEN 'schema'
+
+                 WHEN partkey IS NOT NULL
+                 THEN 'distributed'
+
+                 WHEN repmodel = 't'
+                 THEN 'reference'
+
+                 ELSE 'distributed'
+           END AS citus_table_type,
+
+           coalesce(column_to_column_name(logicalrelid, partkey), '<none>')
```

Comment:
I am unsure if it looks better if we capitalize `coalesce` call here.

Feel free to ignore this suggestion.

```suggestion
           COALESCE(column_to_column_name(logicalrelid, partkey), '<none>')
```